docs/working_with_resources.md

```diff
@@ -0,0 +1,61 @@
+# Working with Resources
+
+*This document is aimed at users who have worked through some of the examples,
+and who want to learn more about using kubectl to manage resources such
+as pods and services.  Users who want to access the REST API directly,
+and developers who want to extend the kubernetes API should 
+refer to the [api conventions](../docs/working_with_resources.md) and
```

Comment:
api conventions links to this document. It should link to ../docs/api-conventions.md.